File: Solutions/javascript/SetupSolutions/SetupSolutions.js
const fs = require("fs");
const path = require("path");

module.exports = {
    getPuzzleInput: (year, day) => {
        const filePath = path.join(__dirname,"..", "..", "..", "Inputs", year, day);
        console.log(filePath);
        const allContents = fs.readFileSync(filePath, 'utf-8');
        const arr = allContents.split(/\r?\n/);
    
        return arr;
    }

}

File: Solutions/Golang/SetupSolutions/SetupSolutions.go
package main

import (
	"fmt"
	"log"
	"os"
	"path/filepath"
)

func GetPuzzleInput() {
	file := filepath.Join("..", "..", "..", "Inputs", "")
	content, err := os.ReadFile(file)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Println(string(content))
}
